resources/views/participant/welcome.blade.php

```diff
@@ -1,11 +1,57 @@
 @extends('layouts.participant_master')
 
 @section('content')
-    <div class="title m-b-md">
-        Spot The Bug Challenge
-    </div>
     <p>Logged in as {{Auth::guard('participant')->user()->email}}</p>
 
+    <div id="TopDiv">
```

Comment:
titels mogen in lowercase. Indien meer dan 1 woord kun je een streepje er tussen zetten